homework/index.js

```diff
@@ -99,12 +99,11 @@
       });
   }
 
+  // I think i forgot to delete .sort form this function. it been used already in main:
   function renderOptionElements(repos, select) {
-    repos
-      .sort((a, b) => a.name.localeCompare(b.name))
-      .forEach((repo, index) => {
-        createAndAppend('option', select, { text: repo.name, value: index });
-      });
+    repos.forEach((repo, index) => {
```

Comment:
I don't know what happened here. You seem to have removed the sorting here, but in the final PR it seems to reappear.

Reply:
Yes, it is wrong.
It seems like I forgot to update my other local copy of the project where I test the amendments before transferring them to the main project folder.
I should be more organized on my work.